.github/workflows/ci.yml

```diff
@@ -59,6 +59,11 @@ jobs:
         cd diff-java-tool
         ./gradlew clean build
 
+    - name: Execute diff-java-tool jar
+      run: |
+        cd diff-java-tool
+        java -jar build/libs/diff-java-tool-*.jar --help
```

Comment:
after execution of this command  supposed to be  print of this
`LOGGER.error("Error: invalid command line arguments!");`

but Logger is not configured to print all to console so it is not printed.

please configure logger to print to standard output.

Reply:
lack of logger will block integration to workflow as execution will be silent, and if any problem happens, will not see any details.